connection.js

```diff
@@ -1369,6 +1278,27 @@ function pluralFamilyStringToSingularString(plural) { /* {{{ */
     return pluralToConvert;
 } /* }}} */
 
+function rankedUnrankedGamesString(argNameString) { /* {{{ */
+    if (argNameString.includes("ranked") && !argNameString.includes("unranked")) {
+    // for example "boardsizesranked"
+        return "for ranked games";
+    } else if (argNameString.includes("unranked")) {
+    // for example "boardsizesunranked"
+        return "for unranked games";
+    } else {
+    // for example "boardsizes"
+        return "";
+    }
+} /* }}} */
+
+function minmaxStringList(argNameString) { /* {{{ */
+    if (argNameString.includes("min")) {
+        return ["minimum", "increase", "low"];
```

Comment:
You can add names to the elements by returning an object here instead of an array.

```Js
return ({min: "minimum", inc: "increase", low: "low"})
```
And the later call `minmax.min` instead of `minmaxList[0]}`.

Reply:
indeed !
i didnt know very well how to return a new object, it was really easy in fact lol